### PR TITLE
Sample notebook for remote recognizer (using Text Analytics)

### PR DESCRIPTION
**Pull request by omri374**
Reusing the text analytics example code to create an interactive notebook.

### CODE REVIEW

File: docs/samples/python/Integrating with external services.ipynb [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    "Text Analytics supports multiple PII entity categories. The Text Analytics service\n",
+    "runs a predictive model to identify and categorize named entities from an input\n",
+    "document. The service's latest version includes the ability to detect personal (PII)\n",
+    "and health (PHI) information. A list of all supported entities can be found in the\n",
+    "[official documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/named-entity-types?tabs=personal).\n",
+    "\n",
+    "### Prerequisites\n",
+    "To use Text Analytics with Preisido, an Azure Text Analytics resource should\n",
+    "first be created under an Azure subscription. Follow the [official documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-call-api?tabs=synchronous#create-a-text-analytics-resource)\n",
+    "for instructions. The key and endpoint, generated once the resource is created, should replace the placeholders `YOUR_TEXT_ANALYTICS_KEY>` and `<YOUR_TEXT_ANALYTICS_ENDPOINT>` in this notebook, respectively. \n",
```
Comment (SharonHart) [inline docs/samples/python/Integrating with external services.ipynb:30]:
```suggestion
    "for instructions. The key and endpoint, generated once the resource is created, should replace the placeholders `<YOUR_TEXT_ANALYTICS_KEY>` and `<YOUR_TEXT_ANALYTICS_ENDPOINT>` in this notebook, respectively. \n",
```

File: docs/samples/python/Integrating with external services.ipynb [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    "## Text Analytics Recognizer\n",
+    "In this example we will use the [`TextAnalyticsRecognizer`](text_analytics/example_text_analytics_recognizer.py) sample implementation. This class extends Presidio's [Remote Recognizer](https://microsoft.github.io/presidio/analyzer/adding_recognizers/#creating-a-remote-recognizer) for calling the Text Analytics service REST API. For an additional example of a remote recognizer, see the [ExampleRemoteRecognizer](example_remote_recognizer.py) sample."
```
Comment (SharonHart) [inline docs/samples/python/Integrating with external services.ipynb:32]: Either link the docs site, or github, not sure what's best, I guess github?

File: docs/samples/python/Integrating with external services.ipynb [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+   "execution_count": null,
+   "id": "permanent-samuel",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "analyzer = AnalyzerEngine()\n",
+    "analyzer.registry.add_recognizer(text_analytics_recognizer)\n",
+    "\n",
+    "results = analyzer.analyze(\n",
+    "    text=\"David is 30 years old. His IBAN: IL150120690000003111111\", language=\"en\"\n",
```
Comment (SharonHart) [inline docs/samples/python/Integrating with external services.ipynb:122]: The return of David 😆

**Review by navalev — DISMISSED** [review]
looks good, tested against a text analytics service.

File: docs/samples/python/Integrating with external services.ipynb (hunk 2 above)
Comment (omri374) [reply docs/samples/python/Integrating with external services.ipynb:32]: Which link do you refer to?

Comment (SharonHart) [reply docs/samples/python/Integrating with external services.ipynb:32]: [Remote Recognizer](https://microsoft.github.io/presidio/analyzer/adding_recognizers/#creating-a-remote-recognizer
vs
[ExampleRemoteRecognizer](example_remote_recognizer.py).

**Review by SharonHart — DISMISSED** [review]

**Review by navalev — APPROVED** [review]

File: docs/samples/python/Integrating with external services.ipynb (hunk 2 above)
Comment (omri374) [reply docs/samples/python/Integrating with external services.ipynb:32]: Added a new link to the GH version of the file (already have reference to the docs right after).

File: docs/samples/python/Integrating with external services.ipynb (hunk 3 above)
Comment (omri374) [reply docs/samples/python/Integrating with external services.ipynb:122]: This works even with David involved :) so leaving it here.

**Review by navalev — DISMISSED** [review]

**Review by navalev — APPROVED** [review]

**Review by SharonHart — APPROVED** [review]

**Review by navalev — APPROVED** [review]

**Pull request merged**